Library/Formula/libodfgen.rb

```diff
@@ -1,5 +1,5 @@
 class Libodfgen < Formula
-  homepage "http://sourceforge.net/p/libwpd/wiki/libodfgen/"
+  homepage "https://sourceforge.net/p/libwpd/wiki/libodfgen/"
```

Comment:
Let's close it, it's not that important. (it may be the 'SSL Everywhere' Safari extension in action, though it works also when I disable it. On Firefox it redirects to plaintext.)